docs/static/releasenotes.asciidoc

```diff
@@ -65,6 +66,75 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
 
+[[logstash-7-17-6]]
+=== Logstash 7.17.6 Release Notes
+
+[[notable-7-17-6]]
+==== Notable issues fixed
+
+* Support Ironbank Docker build https://github.com/elastic/logstash/pull/14298[#14298]
+* Fix artifact path for integration with ubireleaser https://github.com/elastic/logstash/pull/14319[#14319]
```

Comment:
Same as above

Reply:
Should I delete ^^ these two, and remove the section head?  OR, if you want to keep the items, I could remove the "notable" section head.